src/utils/helpers.js

```diff
@@ -0,0 +1,16 @@
+import { isNil } from 'lodash';
```

Comment:
the name is not the most original name on the world, but my imagination got me nothing better :) feel free to come with better naming ;)